File: examples/tvos/objc/DownloadCache/Repository.m
//
//  Repository.m
//  RealmExamples
//
//  Created by Katsumi Kishikawa on 11/19/15.
//  Copyright © 2015 Realm. All rights reserved.
//

#import "Repository.h"

@implementation Repository

+ (NSString *)primaryKey {
    return @"identifier";
}

+ (NSArray<NSString *> *)requiredProperties {
    return @[@"identifier"];
}

@end

File: examples/tvos/swift/DownloadCache/Repository.swift
//
//  Repository.swift
//  RealmExamples
//
//  Created by kishikawakatsumi on 11/20/15.
//  Copyright © 2015 Realm. All rights reserved.
//

import UIKit
import RealmSwift

class Repository: Object {
    dynamic var identifier = ""
    dynamic var name: String?
    dynamic var avatarURL: String?

    override static func primaryKey() -> String? {
        return "identifier"
    }
}
